scripts/wgcna_analysis.R

```diff
@@ -0,0 +1,125 @@
+
+
+library(WGCNA);
+library(stringi);
+library(rjson)
+
+options(stringsAsFactors = FALSE);
+
+imgDir = Sys.getenv("GENERATED_IMAGE_DIR")
+# load expression data **assumes from json files row(traits)(columns info+samples)
+# pass the file_path as arg
+# pass the file path to read json data
+
+args = commandArgs(trailingOnly=TRUE)
+
+if (length(args)==0) {
+  stop("Argument for the file location is required", call.=FALSE)
+} else {
+  # default output file
+  json_file_path  = args[1]
+}
+
+inputData <- fromJSON(file = json_file_path)
+
+
+# parse the json data input
+
+minModuleSize <-inputData$minModuleSize
+
+TOMtype <-inputData$TOMtype
+
+corType <-inputData$corType
+# 
```

Comment:
**nitpick:** Remove this.

Reply:
Done here commit: https://github.com/genenetwork/genenetwork3/pull/38/commits/6f25b8e2b1d1a34c054d325b1c37b303529b8827